src/MomentoCache.ts

```diff
@@ -212,6 +213,7 @@ export class MomentoCache {
     return [
       new HeaderInterceptor(headers).addHeadersInterceptor(),
       ClientTimeoutInterceptor(this.requestTimeoutMs),
+      new RetryInterceptor().addRetryInterceptor(),
```

Comment:
do you want to maybe leave a comment here about how we eventually want to make this configurable but we need to do API design first?

Reply:
Yes will add comment good call